src/disease/query.py

```diff
@@ -32,6 +31,42 @@
 _logger = logging.getLogger(__name__)
 
 
+def create_concept_mapping(
```

Comment:
Wasn't sure the best place to put this... Debated putting this in `schemas.py` since it relies on enums/mappings defined there.

Reply:
does it get used outside query? it can probably be given a leading underscore for privacy if not

Reply:
@jsstevenson So I import this in therapy normalizer https://github.com/cancervariants/therapy-normalization/blob/issue-456/src/therapy/query.py#L10 so we can create mappings for indications

Reply:
works for me

Reply:
maybe in schemas then